.github/CODEOWNERS

```diff
@@ -9,6 +9,7 @@
 /rzup/                                 @risc0/platform @hmrtn
 /web/                                  @risc0/platform @nahoc
 /website/                              @risc0/devrel @pdg744
+/examples/                             @risc0/devrel @risc0/demos
```

Comment:
```suggestion
/examples/                             @risc0/devrel @risc0/demos @risc0/platform
```

Also could we sort the dirs?

Reply:
added comments to sections and did a sort ascending. LMK if that is what you wanted?

Reply:
https://github.com/risc0/risc0/blob/86dd3dba48d601316a239f97fae34b56162af1c4/.github/CODEOWNERS

Reply:
Note that there is a global requirement on platform team already :
![image](https://github.com/risc0/risc0/assets/35669742/b8b164ec-0537-496e-9c47-2ed8cf0acc17)